opentelemetry-otlp/src/exporter/tonic/logs.rs

```diff
@@ -65,9 +66,11 @@ impl LogExporter for TonicLogsClient {
             None => return Err(LogError::Other("exporter is already shut down".into())),
         };
 
+        // TODO: Avoid cloning when logdata is borrowed?
```

Comment:
trying to understand the TODO here - if the data is borrowed or not, does it matter for the exporter? Isn't it serializing the data inside export() call itself, and no longer need to keep the original data around....

Reply:
As of now, we are always cloning which can be avoided. Have updated the TODO to reflect that. Will address this separately.